From 0740e9878bae9a61775ba6c4acadf6dbc73ffb21 Mon Sep 17 00:00:00 2001
From: CI Assistant <yossi.ilkanaev@intel.com>
Date: Mon, 19 Nov 2018 15:08:25 +0100
Subject: [PATCH] Merge pull request #278 in SW_UGW/linux from
 bugfix/DRVLIB_SW-961-change-BM-min-guaranteed-for-policy0 to master

* commit '47ce9952d45f5214e37ae02fd5d2fef46d4fbb92':
  DRVLIB_SW-961-change-BM-min-guaranteed-for-policy0 to 0x40 in cqm driver
---
 drivers/net/ethernet/lantiq/cqm/falconmx/cqm.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/drivers/net/ethernet/lantiq/cqm/falconmx/cqm.c b/drivers/net/ethernet/lantiq/cqm/falconmx/cqm.c
index be86abe92fcf..b9a37756743d 100644
--- a/drivers/net/ethernet/lantiq/cqm/falconmx/cqm.c
+++ b/drivers/net/ethernet/lantiq/cqm/falconmx/cqm.c
@@ -3567,7 +3567,7 @@ static int conf_bm(struct cqm_data *pdata)
 		switch (i) {
 		case CQM_SIZE0_BUF_SIZE:
 			p_param[i].max_allowed = cqm_ctrl->fmx_pool_ptrs[i];
-			p_param[i].min_guaranteed = cqm_ctrl->fmx_pool_ptrs[i];
+			p_param[i].min_guaranteed = 0x40;
 			break;
 		case CQM_SIZE1_BUF_SIZE:
 			p_param[i].max_allowed = (cqm_ctrl->fmx_pool_ptrs[i]
